style.css

```diff
@@ -1,4 +1,622 @@
+    transform: translateY(-2px);
+    width: 98px;
+  }
+
+  .header-menu {
+    background-color: #ededed;
+    height: 69px;
+    margin: 0;
+    right: 0;
+    width: 70px;
+  }
+
+  .header-menu__line::before {
+    content: '';
+    transform: translateY(-9px);
+  }
+
+  .header-menu__line,
+  .header-menu__line::before {
+    display: block;
+    height: 3px;
+    left: 50%;
+    margin-left: -14px;
+    top: 52%;
+    width: 27px;
+  }
+
+  .language,
+  .collection__img,
+  .collection__description,
+  .collection__btn,
+  .range,
+  .socials-list,
+  .scroll {
+    display: none;
+  }
+
+  .collection {
+    background-image: url('./images/pete-bellis-432867-unsplash-mob.png');
+    background-position: center center;
+    background-size: cover;
+    height: 870px;
+    object-fit: cover;
+    padding: 0;
+    width: 100%;
+  }
+
+  .collection-content {
+    width: 100%;
+  }
+
+  .collection__title {
+    font-size: 54px;
+    line-height: 56px;
+    margin-left: 5vw;
+    margin-top: 670px;
+    text-align: center;
+  }
+
+  .collection__title::after {
+    content: "";
+    position: absolute;
+    width: 38px;
+    height: 38px;
+    border-right: 5px solid #000;
+    border-top: 5px solid #000;
+    margin-left: 40%;
+    margin-top: 9%;
+    transform: rotate(135deg);
+  }
+}
+
+@media (min-width: 320px) and (max-width: 540px) {
+  * {
+    box-sizing: border-box;
+  }
+
+  html,
+  body {
+    margin: 0 auto;
+    max-width: 540px;
+    min-width: 320px;
+    overflow-x: hidden;
+    width: 100%;
+  }
+
+  .main-page {
+    background: none;
+    padding: 0;
+  }
+
+  .header,
+  .collection {
+    margin: 0;
+    width: 100%;
+  }
+
+  .header {
+    background-color: #fff;
+    display: flex;
+    font-size: 0;
+    height: 69px;
+    justify-content: flex-start;
+  }
+
+  .header__logo {
+    background-image: url('./images/LOCO.png');
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: fixed;
+    display: flex;
+    height: 37px;
+    margin-left: 23px;
+    transform: translateY(-2px);
+    width: 98px;
+  }
+
+  .header-menu {
+    background-color: #ededed;
+    height: 69px;
+    margin: 0;
+    right: 0;
+    width: 70px;
+  }
+
+  .header-menu__line::before {
+    content: '';
+    transform: translateY(-9px);
+  }
+
+  .header-menu__line,
+  .header-menu__line::before {
+    display: block;
+    height: 3px;
+    left: 50%;
+    margin-left: -14px;
+    top: 52%;
+    width: 27px;
+  }
+
+  .language,
+  .collection__img,
+  .collection__description,
+  .collection__btn,
+  .range,
+  .socials-list,
+  .scroll {
+    display: none;
+  }
+
+  .collection {
+    background-image: url('./images/pete-bellis-432867-unsplash-mob.png');
+    background-position: center;
+    background-size: cover;
+    display: flex;
+    margin: 0;
+    min-height: 529px;
+    object-fit: cover;
+    padding: 0;
+    width: 100%;
+  }
+
+  .collection-content {
+  	width: 100%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5658198)